scripts/testnet/include/config.sh

```diff
@@ -166,6 +166,17 @@ generateProxyObserverList() {
       let OBSERVER_INDEX++
     done
   done
+  # Start Meta Observers
+  for OBSERVER_IN_SHARD in `seq $META_OBSERVERCOUNT`; do
```

Comment:
Rename `OBSERVER_IN_SHARD` to `METAOBSERVER`. It doesn't change the function, but it's clearer.

Reply:
done.